CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#242](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/242))  
 - `opentelemetry-instrumentation-flask` Do not emit a warning message for request contexts created with `app.test_request_context`
   ([#253](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/253))
+- `opentelemetry-instrumentation-requests`, `opentelemetry-instrumentation-urllib` Fix span name callback parameters
+- ([#259](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/259))
```

Comment:
Note that there's an extra dash on line 43 here.

Reply:
Probably will be fixed in a new pr